docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -677,14 +677,33 @@ CAUTION: Use this tool at your own risk, since there is a chance of potential le
 
 === How to use an encrypted string in VIVIDUS
 
-. Add the secret key/text to the project properties as the value of:
+. Set the password in one of the following ways:
++
+.. Pass the key with its password-value into a test run using command line:
++
+[source,shell]
+----
+./gradlew runStories -Pvividus.encryptor.password=MYPAS_WORD
+----
++
+.. Add the password to the system project properties as the value of:
 +
 [source,properties]
 ----
 system.vividus.encryptor.password=MYPAS_WORD
 ----
 +
-IMPORTANT: This value must be kept secret and must not be committed to version control system.
+.. Set via OS environment variable `VIVIDUS_ENCRYPTOR_PASSWORD`:
++
+[source,shell]
+----
+export VIVIDUS_ENCRYPTOR_PASSWORD=MYPAS_WORD
+----
```

Comment:
is it applicable for power shell?

Reply:
> is it applicable for power shell?

Based on the [documentation ](https://zsh.sourceforge.io/Doc/Release/Shell-Builtin-Commands.html), yes

Reply:
> is it applicable for power shell?

No, this is only for "Z shell"

Reply:
> maybe give some general guidelines to not couple this case to particular platform

Yes, but in this case it's better to do it [here](https://docs.vividus.dev/vividus/latest/tests-configuration.html#_externalized_configuration)